app/models/flood_risk_engine/enrollment.rb

```diff
@@ -42,6 +42,10 @@ def state_machine
       to: :state_machine
     )
 
+    def state
+      self.step
+    end
+
```

Comment:
Actually, the easiest way is to add it to the list of methods passed through via delegate.